lib/internal/test_runner/test.js

```diff
@@ -655,29 +675,37 @@ class Test extends AsyncResource {
       this.parent.processReadySubtestRange(false);
       this.parent.processPendingSubtests();
     } else if (!this.reported) {
+      const {
+        diagnostics,
+        harness,
```

Comment:
The callers of this used to call `this.root.harness` are they the same?

Reply:
Yes, they are the same.